src/main/java/codersafterdark/reskillable/base/LevelLockHandler.java

```diff
@@ -42,44 +43,27 @@
     private static Map<Class<?>, List<Class<? extends LockKey>>> lockTypesMap = new HashMap<>();
     private static Map<LockKey, Set<FuzzyLockKey>> fuzzyLockInfo = new HashMap<>();
     private static String[] configLocks;
+    private static Map<String, Class<? extends LockKey>> lockNameMap = new HashMap<>();
+
+    static {
+        registerLockKeyName("itemstack", ItemInfo.class);
+        registerLockKeyName("mod", ModLockKey.class);
+        registerLockKeyName("nbt", GenericNBTLockKey.class);
+    }
 
     public static void loadFromConfig(String[] configValues) {
```

Comment:
Can be removed

Reply:
Why should this be able to be removed?
This is needed for the json parser to deserialize the key name.

Reply:
The `loadFromConfig(String[] configValues)` method. Github in its infinite wisdom decided to also display those lines of your code.